### DISCUSSION TITLE
Packages/com.unity.textmeshpro/api/TMPro.TextMeshProUGUI

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#424] [Notes]
# Packages/com.unity.textmeshpro/api/TMPro.TextMeshProUGUI

https://docs.unity3d.com/Packages/com.unity.textmeshpro@latest/index.html?subfolder=%2Fapi%2FTMPro.TextMeshProUGUI.html

**Comment by vertxxyz** [comment]
If you're having issues parsing from `text`, and you're using an TMP input field, do not reference the underlying TextMeshPro object, reference the [`TMP_InputField`](https://docs.unity3d.com/Packages/com.unity.textmeshpro@latest/index.html?subfolder=/api/TMPro.TMP_InputField.html) itself.
The child `TextMeshProUGUI` uses a zero-width space for layout purposes, and should not be referenced.
